appveyor.yml

```diff
@@ -16,13 +16,26 @@ environment:
 install:
   - appveyor DownloadFile http://slproweb.com/download/Win32OpenSSL-1_0_1L.exe
   - Win32OpenSSL-1_0_1L.exe /silent /verysilent /sp- /suppressmsgboxes
+  - appveyor DownloadFile "http://sunet.dl.sourceforge.net/project/nsis/NSIS 2/2.46/nsis-2.46-setup.exe" -FileName nsissetup.exe
```

Comment:
You could replace this with a chocolatey package I think:
https://chocolatey.org/packages/nsis

`cinst nsis` should work.